.github/workflows/deploy-to-aks.yaml

```diff
@@ -0,0 +1,157 @@
+name: Create AKS cluster, deploy kubeflow-mlflow Terraform solution and run UATs
+on:
+  workflow_dispatch:
+    inputs:
+      k8s_version:
+        description: 'Kubernetes version to be used for the AKS cluster. Make sure that the corresponding K8s version is supported by the cloud.'
+        required: false
+      uats_branch:
+        description: 'Branch to run the UATs from e.g. main or track/1.9. By default, this is defined by the dependencies.yaml file.'
+        required: false
+  schedule:
+    - cron: "17 02 * * 1"
+
+  # TODO: remove this before merging, this is here in order to confirm that the workflow works
+  # Since no inputs are given, this simulates what will happen when scheduled
+  pull_request:
+
+jobs:
+  deploy-solution-to-aks:
+    name: Deploy CKF + MLFlow solution to AKS
+    runs-on: ubuntu-22.04
+    env:
+      AZURE_CORE_OUTPUT: none
+      PYTHON_VERSION: "3.8"
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+
+      - name: Run YAML to Github Output Action
+        id: yaml-output
+        uses: christian-ci/action-yaml-github-output@v2
+        with:
+          file_path: ".github/dependencies.yaml"
+
+      - name: Update ENV variables from inputs if available
+        run: |
+          K8S_VERSION=${{ inputs.k8s_version || env.K8S_VERSION }}
+          echo "K8S_VERSION=${K8S_VERSION}" >> $GITHUB_ENV
+          UATS_BRANCH=${{ inputs.uats_branch || env.UATS_BRANCH }}
+          echo "UATS_BRANCH=${UATS_BRANCH}" >> $GITHUB_ENV
+
+      # Remove once https://github.com/canonical/bundle-kubeflow/issues/761
```

Comment:
does it make more sense to link https://github.com/canonical/charmed-kubeflow-uats/issues/79 here?

Reply:
Fixed